### ISSUE TITLE
Cannot read property 'forEach' of undefined when iterating html files

### ISSUE THREAD

**Issue by skoblenick** [import-js/eslint-plugin-import#2210]
Linting all files with a global rule results in:

```
Linting "demo"...
Cannot read property 'forEach' of undefined
Occurred while linting ./apps/demo/src/app/app.component.html:1
```

I have a sample repo that reproduces the issue here: https://github.com/skoblenick/eslint-plugin-import-issue

I realize I could technically move the plugin into an `override` that is specific to the `.ts` files however it shouldn't be necessary when the code could just be more defensive.  I could imagine a use case where I may have preprocessor working on imports within a html or template file (like an `hbs`) and I want them ordered (hence a lint check).

**Comment by skoblenick** [comment]
So I see a fix in a third part repo but I don't see this merged into `main`

**Comment by skoblenick** [comment]
@ljharb did you submit a PR back to the origin from your fork?

**Comment by ljharb** [comment]
@skoblenick that's inaccurate; the commit is indeed in main, and it's not possible for the commit to have closed this issue unless it landed in main in *this repo*, so I'm not sure what your confusion is.

**Comment by ljharb** [comment]
@Dima-Astreyko the fix is included in v2.25.4+.
